client/src/App.js

```diff
@@ -1,26 +1,54 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import axios from "axios";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+//component imports
+import PlayerCard from "./components/PlayerCard";
+import Navbar from "./components/Navigation";
+import Toggle from "./components/toggle";
+
+class App extends React.Component {
+  //create class component and state
+  state={
+    player: [],
+  }
+
+  componentDidMount() {
+    //add axios to call
+    //run npm i on the root folder to start localhost: 5000
+    axios
+      .get(`http://localhost:5000/api/players`)
+      .then(res =>{
+        //console.log res, and continue until desired data is found
+        console.log(`the response form the axios call is:`, res.data)
+        //console.log typeof res.data
+        console.log(`res.data is a/an`, typeof res.data)
+        //set res.data to state
+        this.setState({
+          player: res.data
+        })
+      })
+  }
+
+  render(){
+    return (
+      <div className="App">
+        <Navbar />
+        <header className="App-header">
+          <h1>Test, hello world!</h1>
+          <Toggle />
+          <div className="card-container">
+
+            <PlayerCard key={this.state.player} player={this.state.player}/>
+            {this.state.player.map((player, key) => {
```

Comment:
how can `this.state.player` be both map'ed over on 43 and send in as a key & player prop on 42?